DTIPrep/DTI/DTI.pm

```diff
@@ -180,14 +180,14 @@ sub getAnatFile {
 =head3 getRawDTIFiles($nativedir, $DTI_volumes)
 
 Function that parses files in the native MRI directories and fetches DTI files.
-This function will also concatenate together multipleDTI files if DTI 
-acquisition performed accross several DTI scans.
+This function will also concatenate together multiple DTI files if DTI
+acquisition performed across several DTI scans.
```

Comment:
THE acquisition WAS performed across ...

Reply:
done